README.md

```diff
@@ -4,16 +4,24 @@ nxdk is a software development kit for the original Xbox. nxdk is a revitalizati
 It is maintained by the [XboxDev](https://github.com/XboxDev/XboxDev) community.
 
 Notable features:
+- Portable toolchain that works on modern versions of Windows, macOS and Linux.
 - No complicated cross-compiling or big library dependencies! Builds with `make` and just needs standard tools and llvm.
```

Comment:
This sounds weird and should be touched.

It can be misinterpreted as: "nxdk is not a cross-compiler". I also don't think that "make" is a good feature, and I don't think "standard tools" aren't really necessary. llvm is the only interesting bit, but could be phrased differently.

Could be something like "Interoperability with standard tooling like gcc and Microsoft Visual C++!" (which I'm not sure about, because nobody tested it yet). Or it could be "LLVM / clang based, for interoperability with other programming languages" (which I'm also not sure about, because nobody tried that yet either; my attempts with C# were unsuccessful, but largely due to shortcomings in the projects I tried).

I think I'd just remove this sentence altogether, or rephrase it to what it seems to imply: "Easy installation."